workspaces/ballerina/ballerina-extension/src/features/project/cmds/cmd-runner.ts

```diff
@@ -98,11 +99,13 @@ let terminal: Terminal;
 
 export function runCommand(file: BallerinaProject | string, executor: string, cmd: BALLERINA_COMMANDS,
     ...args: string[]) {
+    TracerMachine.startServer();
```

Comment:
`TracerMachine.startServer()` is called multiple times in different execution paths. `runCommandWithConf` function also has `TracerMachine.startServer()` call. May be we can remove it from here.

```suggestion
    
```